### PR TITLE
ClassProperty should accept semicolons

### LINKED ISSUE #104: Add support for class properties of the es7
Relates to https://github.com/jscs-dev/node-jscs/blob/9a3a181ede0fdb748c24feb9bfdba5ac544f6c5b/test/specs/rules/require-semicolons.js#L397

### PR DESCRIPTION
**Pull request by hkosiacki**
This change should prevent errors such as http://stackoverflow.com/questions/39510025/jscs-punctuator

Fixes #104

### CODE REVIEW

**Comment by markelog** [comment]
Sorry, missed this too, could you rebase?

File: test/3rdParty/xfail.json [hunk 1]
```diff
@@ -9,12 +9,8 @@
             "38": "skip",
             "39": "skip",
             "40": "skip",
-            "43": true,
-            "44": true,
-            "45": true,
-            "46": true,
-            "47": true,
-            "48": true,
+            "47": "skip",
```
Comment (graingert) [inline test/3rdParty/xfail.json]: do you need these skips?

**Comment by graingert** [comment]
@hkosiacki @markelog check out https://github.com/cst/cst/pull/161 instead

File: test/3rdParty/xfail.json (hunk 1 above)
Comment (hkosiacki) [inline test/3rdParty/xfail.json]: As far as I can remember, 47 and 48 did hang the tests, I guess your patch in #161 deals with that now.

**Pull request closed without merging**